packages/ai/src/chat/providers.ts

```
@@ -33,7 +33,7 @@ export function initModel({
             providerOptions = {
                 openrouter: { transforms: ['middle-out'] },
             };
-            const isClaude = requestedModel === OPENROUTER_MODELS.CLAUDE_4_SONNET;
+            const isClaude = requestedModel === OPENROUTER_MODELS.CLAUDE_4_SONNET || requestedModel === OPENROUTER_MODELS.CLAUDE_4_5_SONNET || requestedModel === OPENROUTER_MODELS.CLAUDE_3_5_HAIKU;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)